include/deal.II/simplex/grid_generator.h

```diff
@@ -0,0 +1,246 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_simplex_grid_generator_h
+#define dealii_simplex_grid_generator_h
+
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/point.h>
+
+#include <deal.II/grid/tria.h>
+
+#include <vector>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Simplex
+{
+  /**
+   * This namespace provides a collection of functions for generating simplex
+   * triangulations for some basic geometries.
+   */
+  namespace GridGenerator
+  {
+    /**
+     * Create a coordinate-parallel brick from the two diagonally opposite
+     * corner points @p p1 and @p p2. The number of vertices in coordinate
+     * direction @p i is given by <tt>repetitions[i]+1</tt>.
+     *
+     * @note This function connects internally 4/8 vertices to quadrilateral/
+     *   hexahedral cells and subdivides these into 2/5 triangular/
+     *   tetrahedral cells.
+     */
+    template <int dim, int spacedim>
+    void
+    subdivided_hyper_rectangle(Triangulation<dim, spacedim> &   tria,
+                               const std::vector<unsigned int> &repetitions,
+                               const Point<dim> &               p1,
```

Comment:
```suggestion
                               const Point<spacedim> &               p1,
```

Reply:
I applied the documentation suggestions. However, I would like to keep `Point<dim>` because this is the same interface as in the regular `GridGenerator`.